sample.html

```diff
@@ -1641,6 +1655,7 @@ <h4 class="panel-title">Options</h4>
     function sendSerialData() {
         var properties = {
             baudRate: $("#serialBaud").val(),
+            type: $("#serialPlainRadio").prop("checked") ? "PLAIN" : "FILE",
```

Comment:
Radio type input can have a `value=""` attribute on them, so we can pull these directly with a `$("input[name='serialType']:checked").val()` call instead of having to use a ternary like this, especially if we need more types later.